bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRIssueWorkItem.java

```diff
@@ -99,16 +99,17 @@ public Collection<WorkItem> run(Path scratchPath) {
                 .flatMap(uri -> bot.repositories().stream()
                         .flatMap(r -> r.parsePullRequestUrl(uri.toString()).stream()))
                 .filter(Issue::isOpen)
+                .filter(pr -> bot.getPRBot(pr.repository().name()).enableCsr())
                 // This will mix time stamps from the IssueTracker and the Forge hosting PRs, but it's the
                 // best we can do.
-                .map(pr -> new PullRequestWorkItem(pr.repository(), pr.id(), csrIssue.project(), csrIssue.updatedAt(), errorHandler))
+                .map(pr -> new CheckWorkItem(bot.getPRBot(pr.repository().name()), pr.id(), errorHandler, csrIssue.updatedAt(), true, true))
                 .forEach(ret::add);
         return ret;
     }
 
     @Override
     public String botName() {
-        return CSRBotFactory.NAME;
+        return PullRequestBotFactory.NAME;
     }
 
     @Override
```

Comment:
Should probably change the `workItemName` to something like "csrissue".

Reply:
Will change it